guides/hack/11-built-in-types/35-enum-class.md

```diff
@@ -169,6 +169,33 @@ If either `D1`, `D2` or `D3` tries to define a constant named `Age`, there will
 
 Though the `final` keyword is not supported, Enum classes support the [`__Sealed`](../attributes/predefined-attributes#__sealed) attribute. Using `__Sealed`, you can specify which other enum classes, if any, are allowed to extend from your enum class.
 
+
+## Abstract enum classes
+
+Like regular classes, enum classes come in two flavors: a concrete one and an abstract one. An abstract enum class can declare abstract members, where only their type and name are provided.
```

Comment:
```suggestion
Like regular classes, enum classes come in two flavors: concrete and abstract. An abstract enum class can declare abstract members (properties and methods), where only their type and name are provided.
```
I like being explicit when referring to what a member is, because that was confusing to me when I was a new reader.

Reply:
Ok, so I'll be more explicit, but it will be constants. Enum classes don't have properties or methods :)